configs/pma.conf

```diff
@@ -0,0 +1,44 @@
+server {
```

Comment:
Are you sure 2 more `nginx` configs are needed? Can't you just use the pterodactyl one and replace the root / other changes?

Reply:
I believe it is good to leave this separate, after all, it is being installed in `/var/www/phpmyadmin`, not that it is mandatory, but it is good to separate.

Reply:
Oops, I understood the question now 🤣
I'll check it out